### ISSUE TITLE
Model trained sucessfully but not detection result

### ISSUE THREAD

**Issue by Alex-Beh** [castorini/howl#37]
Hello, I managed to trained the mdoel and I have attached my model in the following link: https://drive.google.com/drive/folders/1UE5NydQhPB8-TeLeLeoc7GOaLTMIBwfP?usp=sharing.

But when I run the howl.run.demo, it didn't detect any result. I am wondering whether is my microphone issue or something is wrong when I train my model. Can you give a try on my trained model? Do you mind to share a pretrained model as well?

The train: ConfusionMatrix(tp=0, fp=0, tn=0, fn=0) is always zero. Is it correct?

**Comment by ljj7975** [comment]
I would first make sure `VOCAB` and `INFERENCE_SEQUENCE` are correct.
let me see if I can clean things up and provide pretrained model

**Comment by ljj7975** [comment]
please refer to https://github.com/castorini/howl/pull/39 for the pretrained model.
You might need to append appropriate env vars for your command.

```
VOCAB='[" hey","fire","fox"]' INFERENCE_SEQUENCE=[0,1,2] INFERENCE_THRESHOLD=0 NUM_MELS=40 MAX_WINDOW_SIZE_SECONDS=0.5 python -m howl.run.demo --model res8 --workspace howl-models/howl/hey-fire-fox
```

I tested and found that some of them (especially `NUM_MELS=40`) are absolutely necessary

**Comment by Alex-Beh** [comment]
Hello, the [howl-models](https://github.com/castorini/howl-models) workspace seem doesn't exist yet. Or the link in github is wrong.

**Comment by ljj7975** [comment]
This is the workspace for hey firefox, https://github.com/castorini/howl-models/tree/master/howl/hey-fire-fox

**Comment by Alex-Beh** [comment]
Hello, the pretrained model is fine and I have followed the tutorial and tried to train **fire** keyword spotting detector and it works fine also although sometimes will give false positive result.

I am trying to generate the dataset from Common Voice and train the **hey fire fox** detector. After 200 epoch, the **ConfusionMatrix is (tp=0,fp=1,tn=900,fn=0)**. Then it  won't give any detection result in the demo script. I guess is because there is no coexist of the vocabs in the dataset causing the bad training result. Do you have any suggestion on this?

**Comment by ljj7975** [comment]
I would be pretty surprised if [this instruction](https://github.com/castorini/howl#hey-firefox) doesn't work.

Did you download the datasets in the read me before it was broken?

**Comment by Alex-Beh** [comment]
No I am unable to download the dataset because it was broken. So I am trying the instruction [here](https://github.com/castorini/howl#preparing-a-dataset) to generate the dataset for "Hey Fire Fox" however it doesn't work well.

**Comment by ljj7975** [comment]
I see

@daemon can you check the server?
also how were these dataset constructed, same as in this [instruction](https://github.com/castorini/howl#preparing-a-dataset)?

**Comment by Alex-Beh** [comment]
Yes, I am following the instruction. Below shown the first two commands to generate positive and negative dataset for the hey firefox.

```
VOCAB='["hey","fire","fox"]' INFERENCE_SEQUENCE=[0,1,2] DATASET_PATH=data/heyfirefox-positive python3.6 -m howl.run.create_raw_dataset --negative-pct 0 -i /howl/cv-corpus-5.1-2020-06-22/en --positive-pct 100

VOCAB='["hey","fire","fox"]' INFERENCE_SEQUENCE=[0,1,2] DATASET_PATH=data/heyfirefox-negative python3.6 -m howl.run.create_raw_dataset --negative-pct 5 -i /howl/cv-corpus-5.1-2020-06-22/en --positive-pct 0
```

**Comment by ljj7975** [comment]
did you also do the following steps? alignment? I guess you have. because without doing so, you won't be able to feed it into the howl

**Comment by Alex-Beh** [comment]
Yes I have tried all the steps after generated the dataset.

**Comment by ljj7975** [comment]
This should fix the broken link #43
